frontend/installation-ui/web/micro-ui-internals/packages/modules/pm/src/pages/employee/CreateFieldPlan.js

```
@@ -910,36 +933,80 @@ const CreateFieldPlan = () => {
+    if (key + 1 >= currentKey) return;
     switch (currentKey) {
-      case 1:
-        setShowBackAlert(true);
-        break;
       case 2:
-        setShowBackAlert(true);
+        setCurrentKey(key + 1);
         break;
       case 3:
-        setShowBackAlert(true);
+        const savedActivityAssignments = getDefaultActivityAssignments();
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)